Sources/Signal.swift

```diff
@@ -295,8 +358,28 @@ public final class Signal<Value, Error: Swift.Error> {
 	/// - parameters:
 	///   - generator: A closure that accepts an implicitly created observer
 	///                that will act as an event emitter for the signal.
-	public init(_ generator: (Observer, Lifetime) -> Void) {
-		core = Core(generator)
+	public static func reentrant(_ generator: (Observer, Lifetime) -> Void) -> Signal {
```

Comment:
Should this also use "unserialized"? I wonder if it would be valuable to add the terms "reentrant" and "unserialized" to a glossary somewhere to make it clear when each is used and why?